src/validation/error.messages.ts

```diff
@@ -180,6 +181,25 @@ export enum ErrorMessages {
   CEASED_DATE_BEFORE_FILING_DATE = "Ceased date must be before or on the date of the update statement",
   RESIGNED_ON_BEFORE_FILING_DATE = "Resigned on date must be before or on the date of the update statement",
   DATE_AFTER_MADE_UP_TO_DATE = "The date you provide must be on or before %s",
+  DAY_LENGTH_OF_LEGAL_ENTITY = "The day the legal entity was created must include 1 or 2 numbers",
+  MONTH_LENGTH_OF_LEGAL_ENTITY = "The month the legal entity was created must include 1 or 2 numbers",
+  YEAR_LENGTH_OF_LEGAL_ENTITY = "The year of the date the legal entity was created must include 4 numbers",
+  DAY_OF_CEASED_LEGAL_ENTITY = "The date the legal entity stopped being associated with this overseas entity must include a day",
+  MONTH_OF_CEASED_LEGAL_ENTITY = "The date the legal entity stopped being associated with this overseas entity must include a month",
+  YEAR_OF_CEASED_LEGAL_ENTITY = "The date the legal entity stopped being associated with this overseas entity must include a year",
+  TRUST_LEGAL_ENTITY_CEASED_DATE_BEFORE_TRUST_CREATION_DATE = "The date the legal entity stopped being associated with this trust must be on or after the date the trust was created",
+  TRUST_LEGAL_ENTITY_CEASED_DATE_BEFORE_INTERESTED_PERSON_START_DATE = "The date they ceased being involved in the trust must be on or after the date they became an interested person",
+  INVALID_DATE_OF_LEGAL_ENTITY = "The date the legal entity stopped being associated with this trust must be a real date",
+  DATE_NOT_IN_PAST_OR_TODAY_OF_CEASED_LEGAL_ENTITY = "The date the legal entity stopped being associated with this trust cannot be in the future",
+  DAY_AND_MONTH_OF_CEASED_LEGAL_ENTITY = "The date the legal entity stopped being associated with this trust entity must include a day and month",
+  DAY_AND_YEAR_OF_CEASED_LEGAL_ENTITY = "The date the legal entity stopped being associated with this trust entity must include a day and year",
+  MONTH_AND_YEAR_OF_CEASED_LEGAL_ENTITY = "The date the legal entity stopped being associated with this trustentity must include a month and year",
+  ENTER_DATE_OF_TRUST_CEASED_LEGAL_ENTITY = "Enter the date they ceased being involved in the trust",
```

Comment:
You could perhaps remove the LEGAL_ENTITY postfix on these constant names as the message looks quite generic and might be the same one used in @markpit 's changes?

Reply:
yes see my PR, i've named them _CEASED_TRUSTEE so they are applicable to others, although my messages start 'The date they ...' instead of 'The date the legal entity..'  so maybe they can't be re-used if that text is what is in the ticket.

Reply:
Renamed these.